agenta-web/cypress/e2e/playground.cy.ts

```diff
@@ -0,0 +1,109 @@
+import {randString} from "../../src/lib/helpers/utils"
+
+describe("Playgroynd | Simple prompt", function () {
+    context("when an api key is provided", function () {
+        it("when entering an invalid case format in the app name", () => {
```

Comment:
it(when) they don't match :) 

put this in another context
context(""when entering an invalid case format in the app name",
    it(should fail)

Reply:
All done